src/Linters/BaseLinter.php

```diff
@@ -28,4 +28,26 @@ public function __construct(
   final public function getFile(): string {
     return $this->file;
   }
+
+  // A simple name for the linter, based on the class name
+  <<__Memoize>>
+  public function getLinterName(): string {
+    $r = new \ReflectionClass($this);
+    return $r->getShortName();
```

Comment:
How about:

```Hack
return self::class
  |> Str\split($$, '\\')
  |> C\lastx($$)
  |> Str\strip_suffix($$, 'Linter');
```

?

Should both be more performant (no reflection), and lead to a shorter ignore string